xml/System/Environment.xml

```diff
@@ -676,11 +685,9 @@ goto :EOF
   
  If environment variables are created after the process has started, you can use this method to retrieve only those variables that were created by calling the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%29> method or the <xref:System.Environment.SetEnvironmentVariable%28System.String%2CSystem.String%2CSystem.EnvironmentVariableTarget%29> method with a `target` value of .<xref:System.EnvironmentVariableTarget.Process?displayProperty=nameWithType>.  
   
- To retrieve all environment variables along with their values, call the <xref:System.Environment.GetEnvironmentVariables%2A> method.  
-  
- Environment variable names are not case-sensitive.  
-  
-   
+### On macOS and Linux systems
+
+On macOS and Linux, the `GetEnvironmentVariable(String)` method retrieves the value of any environment variable that is defined within the scope of the dotnet process. Once the dotnet process ends, that environment variable ceases to exist.  .NET Core does not support per-machine or per-user environment variables.   
```

Comment:
This description might be misleading.  A process inherits the environment variables provided to it by the parent process that created it, e.g. if it's launched from the shell, it's going to have whatever environment variables were configured for that user.  I want to make sure we don't imply that the only environment variables available are those explicitly set by a call to Environment.SetEnvironmentVariable.